lib/server.js

```diff
@@ -59,6 +60,9 @@ function serverFactory(core, messageHandler, Logger) {
             logger.info('proxyDHCP server is listening on %s:%s'.format(self.address, self.inPort));
             self.server.setBroadcast(true);
         });
+
+        var DHCPPoller = new IscDhcpLeasePoller({}, {});
+        DHCPPoller.run();
```

Comment:
I'm contemplating with @benbp on moving the ARP poller as an on-core services (for various reasons).  If we do that than we can create a consistent config.json option for both.